README.md

```diff
@@ -1,38 +1,54 @@
 
-The rebalancer has two modes; the single signer mode(testing mode) and the cosmos voting mode, also known as the chain signer mode. The gravity bridge will run the chain signer mode, while the single signer mode can be bootstraped. The section gives an overview on how to bootstrap the testing mode.
+Strategies determine where to invest funds and how to rebalance them in reaction to market events. When developers want to make their own strategy to run on Sommelier, they need two things:
+
+1. A [Cellar contract](docs/Cellarsetup_instructions) containing a `rebalance` function.
+2. [Data Providers](docs/data_providers)
+    
+
+Steward also provides a suite of utility functions for interacting with Sommelier and the Gravity bridge. For instance, the last section shows series of subcommands that allows you to interact with the Sommelier chain via Steward. 
+
+## Getting started with the testing mode
+
+Steward has two modes; the single signer mode(testing mode) and the allocation mode. The Sommelier chain will run the allocation module, while the single signer mode can be bootstraped. The section gives an overview on how to bootstrap the testing mode.
 
 ### Testing mode
 
-These instructions assume that the cellar has been deployed to the target ethereum blockchain.
+These instructions assume that the Cellar has been deployed to the target ethereum blockchain.
 
 #### Setup Configuration File
 The first step is to setup your configuration file.
 
 To generate a configuation file template, run the command below in your terminal:
 
 ```
-cargo run -- print-config
+cargo run --bin steward print-config
 ```
 
-Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed cellar contract.
+Next, create a `toml` file in the root of the application, replacing the default keys in the template displayed in your terminal with your configuration. Make sure to confirm that the token info in your configuration file matches the deployed Cellar contract.
 
-You can create keys or import keys. To create keys, run the command below:
+You can create keys or import keys. To create keys, run the command below. Replace `eth` with `cosmos` if you want to create a `cosmos` key instead of an `eth` key.
+
+```
+cargo run --bin steward -- -c [your_config_file_name.toml] keys eth add [key_name]
```

Comment:
Nit: not a required fix, but the `--` between "steward" and "-c" in these commands is unnecessary.

Reply:
I think it is, it doesn't run without it.

Reply:
This works: 

```
ugochi@ugochi-GF63-Thin-10SC:~/cellar_rebalancer_rs$ cargo run --bin steward -- -c me.toml keys eth add ugochi
warning: hidden lifetime parameters in types are deprecated
```
This doesn't: 

```
ugochi@ugochi-GF63-Thin-10SC:~/cellar_rebalancer_rs$ cargo run --bin steward -c me.toml keys eth add ugoch
error: Found argument '-c' which wasn't expected, or isn't valid in this context
```

Reply:
After looking into it, you're correct here. I was testing it without passing the `-c` config option. The reason is that the binary we're running from bash's perspective is `cargo`, but we're by proxy invoking `steward` with arguments. The `--` indicates the end of `cargo` flags and the binary is then interpreting the rest as if it were running `steward`.

Reply:
nice, will you merge then?

Reply:
👍

Reply:
Sorry about that, I think I forgot which version of the command I was responding to. My bad!